### PR TITLE
XMLBear.py: Add RelaxNG support

### PR DESCRIPTION
**Pull request by danlls**
Add support for RelaxNG schema for validation

Closes https://github.com/coala/coala-bears/issues/1693

### CODE REVIEW

**Comment by jayvdb** [comment]
Please squash your three commits into one commit.

**Comment by danlls** [comment]
Done :)

File: bears/xml2/XMLBear.py [hunk 1]
```diff
@@ -50,17 +50,21 @@ class XMLBear:
...
     def create_arguments(filename, file, config_file,
                          xml_schema: path='',
-                         xml_dtd: path_or_url=''):
+                         xml_dtd: path_or_url='',
+                         xml_relaxng: path=''):
         """
         :param xml_schema: ``W3C XML Schema`` file used for validation.
         :param xml_dtd:    ``Document type Definition (DTD)`` file or
                            url used for validation.
+        :param xml_relaxng:``RelaxNG Schema`` file used for validation.
```
Comment (jayvdb) [inline bears/xml2/XMLBear.py]: Does it support a URL, like `xml_dtd`?

File: bears/xml2/XMLBear.py [hunk 2]
```diff
@@ -50,17 +50,21 @@ class XMLBear:
     @staticmethod
     def create_arguments(filename, file, config_file,
                          xml_schema: path='',
```
Comment (jayvdb) [inline bears/xml2/XMLBear.py]: I also wonder if this might support a URL also?
If so, it should use `path_or_url`

File: bears/xml2/XMLBear.py [hunk 3]
```diff
@@ -50,17 +50,21 @@ class XMLBear:
...
-                         xml_dtd: path_or_url=''):
+                         xml_dtd: path_or_url='',
+                         xml_relaxng: path=''):
```
Comment (jayvdb) [inline bears/xml2/XMLBear.py]: Does it support a URL, like `xml_dtd`?  If so, this should be `path_or_url`, and the docstring updated.

**Comment by jayvdb** [comment]
ack af3cd72

**Comment by jayvdb** [comment]
Also, needs rebase.

**Comment by danlls** [comment]
@jayvdb URL is not supported. Rebased

**Comment by jayvdb** [comment]
reack 83a8ac9

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**